sydent/db/threepid_associations.py

```diff
@@ -80,15 +80,13 @@ def removeAssociation(self, threepid, mxid):
         )
         row = cur.fetchone()
         if row[0] > 0:
-            ts = time_msec()
             cur.execute(
-                "REPLACE INTO local_threepid_associations "
-                "('medium', 'address', 'mxid', 'ts', 'notBefore', 'notAfter') "
-                " values (?, ?, NULL, ?, null, null)",
-                (threepid['medium'], threepid['address'], ts),
+                "DELETE FROM local_threepid_associations "
+                " WHERE medium = ? AND address = ?",
+                (threepid['medium'], threepid['address']),
             )
             logger.info(
-                "Deleting local assoc for %s/%s/%s replaced %d rows",
```

Comment:
Why remove this? And if you do you need to remove the `cur.rowcount` below. Has this been tested?

Reply:
Because we're not replacing rows anymore.

> Has this been tested?

The deleting rows bit yes, this log change no. Whoopsie.

Reply:
You could also log how many we deleted, but thats ok